ismatec/driver.py

```diff
@@ -135,18 +135,18 @@ async def get_runtime(self, channel: int) -> float:
     async def set_runtime(self, channel: int, runtime: float) -> bool:
         """Set the runtime (minutes) of a channel."""
         assert channel in self.channels
-        return self.hw.command(f'{channel}xT{self._time2(runtime, units="m")}')
+        return self.hw.command(f'{channel}xT{self._time2(runtime, units='m')}')
 
     async def get_volume_setpoint(self, channel) -> float:
-        """Get the volume setpoint, in mL, of a channel."""
+        """Get the volume setpoint (mL) of a channel."""
         return float(self.hw.query(f'{channel}v')) / 1000
 
     async def set_volume_setpoint(self, channel, vol: float) -> bool:
         """Set the volume (mL) of a channel."""
         assert channel in self.channels
         return self.hw.command(f'{channel}v{self._volume2(vol)}')
 
-    async def get_rotation(self, channel: int) -> Protocol.Rotation:
+    async def get_rotation(self, channel: int) -> Protocol.Rotation:  # Don't like this
```

Comment:
I take back "Don't like this" after reading the manual. It's a reasonable solution, but it's still one that needs clarification.

We're basically returning a custom data type. It's a textbook case for an enum, but it also needs to be documented. I'd add commentary to the docstring about the return type.

How does `print(pump.get_rotation())` go? Will it return a string saying CLOCKWISE, or will it return some mess like `<Enum Protocol.Rotation>`?